### ISSUE TITLE
x-Zoom drag not working as expected with multiple charts 

### ISSUE THREAD

**Issue by DaveenW** [highcharts/highcharts#12868]
Hello,

When placing multiple charts within a div container that has a fixed size / position smaller than all the charts added up, it becomes impossible to "drag zoom" (using zoomType: 'x') on elements which are not present in the div at first.  

#### Expected behaviour
Be able to 'drag zoom' straight away when the div containing the charts has a fixed size / position.

#### Actual behaviour
Impossible to 'drag zoom' at very first on non visible charts, a quick workaround would be to get the cursor out of the chart and get it back on it.

#### Live demo with steps to reproduce
steps:
1/ Create a div with limited size.
2/ Generate multiple charts within.
3/ Scroll up / down to a hidden chart.
4/ Attempt to click and drag to zoom
-- to resolve it --
5/ move your mouse outside the chart and re enter after
6/ drag zoom works again for visible chart.

Please find a demo there:
https://jsfiddle.net/DaveenW/kf83z7pg/12

Thanks for your suggestions

#### Product version
Highstock: latest version

#### Affected browser(s)
At least: Firefox, Chrome and Edge
Haven't tried on Safari

**Comment by pawelfus** [comment]
Thanks for reporting this issue @DaveenW - it's a regression, v7.2.0 used to work fine.

**Workaround:**
Manually clear `chart.pointer.chartPosition` when mouse enters container: https://jsfiddle.net/BlackLabel/176294kx/

Snippet:
```js
      chart: {
        renderTo: 'test' + i,
        zoomType: 'x',
        events: {
          load: function() {
            const chart = this;
            Highcharts.addEvent(
              chart.container,
              'mouseenter',
              function() {
                chart.pointer.chartPosition = null;
              }
            );
          }
        }
      },
```

*Internal note:*
Bisected to commit 8359afc222

**Comment by DaveenW** [comment]
Thank you for the work around @pawelfus.

However, though the drag zoom works a lot more often when the pointer first enters the chart, it is not a 100% thing.
The likelihood of reproducing the behaviour increases with the number of charts:
https://jsfiddle.net/DaveenW/kf83z7pg/14/

**Comment by pawelfus** [comment]
Hm.. I'm unable to recreate issue in your demo @DaveenW It might be connected to the fact I use osx+touchpad instead of mouse scroll. Does it work for you @DaveenW if you replace `mouseenter` with `mousemove` ?

**Comment by DaveenW** [comment]
Indeed, clearing `chart.pointer.chartPosition` on `mousemove` does the trick perfectly. Wielkie dzięki @pawelfus .

**Comment by pawelfus** [comment]
*Internal note:*
Implement the event above, start with `mouseenter` and see if we can recreate the issue further. This solution most likely fixes also #12601 - after fixing, check and close or leave a note in #12601
